.github/workflows/ci.yml

```diff
@@ -8,62 +8,41 @@ on:
       - main
 
 jobs:
-  check-fmt:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: actions-rust-lang/setup-rust-toolchain@v1
-        with:
-          toolchain: stable
-      - run: cargo fmt --all --check
-      - run: cd nova-benches && cargo fmt --check
-
   check-build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions-rust-lang/setup-rust-toolchain@v1
-        with:
-          toolchain: stable
-          target: riscv32i-unknown-none-elf
-      - run: cargo check --all-features
-      - run: cargo check --all-features --examples
+      - name: Install tooling and use Rust cache
+        uses: ./.github/actions/rust-cache
+      - run: cargo fmt --all --check
       - run: cargo check -p example --target=riscv32i-unknown-none-elf
-      - run: cd nova-benches && cargo check --benches
```

Comment:
Why was `cd nova-benches` dropped?

Reply:
My bad, will add it back asap.

Reply:
fixed in [fc33a1d](https://github.com/nexus-xyz/nexus-zkvm/pull/269/commits/fc33a1dfb94e81140eef538e3cfbe04ff9268c63)